mock-apns/README.md

```diff
@@ -0,0 +1,28 @@
+# APNs Mock Server
+
+## Setup
+Before you run the mock server you need to create a ssl certificate
+```bash
+openssl req -new -x509 -keyout private -out private -nodes
+```
+
+Converse it to DER format
+```bash
+openssl x509 -in private -out cert.crt -outform DER
```

Comment:
This is confusing and is related to my previous comment. You are using 'x509' flag that expects a x509certificate, but then you specify a file named 'private' (that makes you think you are passing a private key).
It works now because you specified 'private' for both certificate and private key (overwriting the key with the certificate), but it is very confusing.